Rear Rider/Rear Rider/ConfigurationOptions/UserConfig.swift

```diff
@@ -85,9 +85,14 @@ class UserConfig: ObservableObject {
     
     // default values. get overwritten on successful load
     var audioFile: String = ConfigOptions.AudioFile.honk.rawValue
-    var lightPattern: String = ConfigOptions.LightPattern.strobe.rawValue
+    var lightPattern: Int = ConfigOptions.LightPattern.strobe.rawValue
     var lightColor: String = "rgb(255,255,255)"
     var lightBrightness: Int = 1
+    
+    var colorRGB: [CGFloat]? // use this for sending rgb values to RPi
```

Comment:
Non-blocking: If we're going to use this to send to the pi (which is great btw; less data transfer), we can get rid of the RGB string entirely. The RGB values are parsed from the string when we load the conf, but if we're going to store the values here instead we can just use this to load the Color too.

Reply:
If you look at my other commits I tried different approaches but for whatever reason the colorpicker would not get updated properly. This is the best solution I could work out for now.

Reply:
That's fine. This was just a thought. Non-blocking means I wouldn't not approve the PR over it